package drizzle;

message MasterList {
  message Record {
    required string hostname = 1;
    optional string username = 2;
    optional string password = 3;
    optional uint32 port = 4;
    optional uint32 connect_retry = 5;
    optional string log_name = 6;
    optional uint64 log_position = 7;
  }

  repeated Record record = 1;
}
